account_analytic_distribution/models/account_analytic_distribution.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# Copyright 2017 Vicent Cubells - <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import _, api, fields, models
+from openerp.exceptions import ValidationError
+
+
+class AccountAnalyticDistribution(models.Model):
+    _name = "account.analytic.distribution"
+    _description = "Analytic distribution"
+    _order = "name asc"
+
+    def _default_company(self):
+        return self.get_default_company()
+
+    name = fields.Char(
+        string='Name',
+    )
+    company_id = fields.Many2one(
+        comodel_name='res.company',
+        string='Company',
+        required=True,
+        default=_default_company,
+    )
+    rule_ids = fields.One2many(
+        string="Distribution rules",
+        comodel_name='account.analytic.distribution.rule',
+        inverse_name='distribution_id',
+    )
+
+    _sql_constraints = [
+        ('name_uniq', 'unique(name, company_id)',
+            _('Distribution name must be unique per Company!')),
+    ]
+
+    @api.model
+    def get_default_company(self):
```

Comment:
Why 2 methods for the same? I supposed that this is due to old API not letting to overwrite, but not you can make only one